halo2_proofs/tests/serialization.rs

```diff
@@ -193,7 +215,7 @@ fn test_serialization() {
                 SingleStrategy<Bn256>,
             >(
                 &verifier_params,
-                pk.get_vk(),
+                &vk, // pk.get_vk()
```

Comment:
leftover?

Reply:
Remove leftover in https://github.com/privacy-scaling-explorations/halo2/pull/361/commits/081a70c199e52cb42780d54ee978c8b361294fb2.